### PR TITLE
Created contact form component for how/get-started pages

### LINKED ISSUE #22: Fix email forms
There's a contact form on the current site, which needs to be replicated before deploying this new website.

Probably something here to use:
https://nextjs.org/docs/guides/building-forms

Contact forms should be sent to post@blanchard.no.

### PR DESCRIPTION
**Pull request by JenyaFTW**
Screenshot 2022-10-14 at 21 53 43

Added contact form component and implemented it in two pages: how, get-started
Implemented endpoint for sending contact form data to email through Sendgrid

Introduced 5 new environment variables:
```
SENDGRID_KEY
CONTACT_TO
CONTACT_FROM
RECAPTCHA_SECRET
NEXT_PUBLIC_RECAPTCHA_SITEKEY
```

fixes #22

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,5 @@
+SENDGRID_KEY=
```
Comment (cobraz) [inline .env.example:1]: Did you add these to Vercel?

File: src/pages/api/contact.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+  if (!validator.isEmail(email)) {
+    return res.status(400).json({ error: 'Invalid email' });
+  }
+
+  if (!validator.isMobilePhone(phone)) {
+    return res.status(400).json({ error: 'Invalid phone number' });
+  }
+
+  sgMail.setApiKey(process.env.SENDGRID_KEY as string);
```
Comment (cobraz) [inline src/pages/api/contact.tsx:64]: We should probably check these variables with something like `envalid`, or use an `invariant` maybe?

**Review by cobraz — APPROVED** [review]
Looks good!

One little nitpicking, but yeah, this looks ready to land ✈️

File: .env.example (hunk 1 above)
Comment (JenyaFTW) [reply .env.example:1]: Yep

File: src/pages/api/contact.tsx (hunk 2 above)
Comment (JenyaFTW) [reply src/pages/api/contact.tsx:64]: Will do

**Pull request merged**